src/styles/rhd-theme/components/_featured-resources.scss

```diff
@@ -1,12 +1,5 @@
 .rhd-c-featured-resources {
-  &-title {
-    justify-content: center;
+  h2.pf-c-title {
```

Comment:
@adelasofia It might be worthwhile to also have the plain `h2` as part of this definition, in the cases where the `.pf-c-title` is not applied. Just a catch-all.

Reply:
Ah good catch, I've updated it 👍